lib/bank_transactions_csv.rb

```diff
@@ -0,0 +1,29 @@
+require 'csv'
+
+class BankTransactionsCsv < RenderCSV
```

Comment:
`BankTransactionsCSV`?

Reply:
it's aligned with `ArticlesCsv`, `OrderCsv` and `OrdergroupsCsv`